packages/utils/src/diff/index.ts

```diff
@@ -21,6 +21,7 @@ import { diffStringsRaw, diffStringsUnified } from './printDiffs'
 import type { DiffOptions } from './types'
 
 export type { DiffOptions, DiffOptionsColor } from './types'
+export type { PrettyFormatPlugin }
```

Comment:
Is it necessary to re-export from `@vitest/utils`?
I see you already have same `export type SnapshotSerializer = PrettyFormatPlugin` from `@vitest/snapshot`, so maybe you can use that for internal purpose.

Reply:
Nice caught! I reused the `SnapshotSerializer` type right now, thanks for your thorough review.